src/libraries/System.Runtime.InteropServices.RuntimeInformation/tests/PlatformVersionTests.cs

```diff
@@ -0,0 +1,242 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Collections.Generic;
+using Xunit;
+
+namespace System.Runtime.InteropServices.RuntimeInformationTests
+{
+    public class PlatformVersionTests
+    {
+        public static IEnumerable<object[]> AllKnownOsPlatforms()
+        {
+            yield return new object[] { OSPlatform.Windows };
+            yield return new object[] { OSPlatform.Linux };
+            yield return new object[] { OSPlatform.OSX };
+            yield return new object[] { OSPlatform.Browser };
```

Comment:
Shouldn't we add other OS versions with this PR:
```csharp
        // Existing properties
        // public static OSPlatform FreeBSD { get; }
        // public static OSPlatform Linux { get; }
        [EditorBrowsable(EditorBrowsableState.Never)]
        public static OSPlatform OSX { get; }
        // public static OSPlatform Windows { get; } //existing
        public static OSPlatform Android { get; }
        public static OSPlatform iOS { get; }
        public static OSPlatform macOS { get; }
        public static OSPlatform tvOS { get; }
        public static OSPlatform watchOS { get; }
```

Reply:
@adamsitnik has that as a separate PR: #39064

Reply:
Oops, sorry didn't know that, thanks